src/packages/core/protocol/account.ts

```diff
@@ -65,6 +74,27 @@ export class Account {
 	/** Public key HEX Buffer */
 	private publicKeyBuffer: Buffer = Buffer.from([]);
 
+	/* Gets the balance of an address (Testnet only for now) */
+	public static async getBalance(address: string): Promise<Object> {
```

Comment:
type the returned object as `Promise<ApiBalance>` or even a more specific protocol interface that inherits from it.

Reply:
Updated